sophie-kaleba/assignment3/BadGridFileException.java

```diff
@@ -0,0 +1,10 @@
+
+public class BadGridFileException extends Exception {
+	
+	private static final long serialVersionUID = 1L;
```

Comment:
This is a habit some people do if they are doing Java serialization, but it isn't really necessary here. Putting it in suggests you don't really know what it is for - if you use this you should be prepared to explain it (eg in an interview)